receiver/postgresqlreceiver/metadata.yaml

```diff
@@ -19,49 +19,49 @@ attributes:
 metrics:
   postgresql.blocks_read:
     description: The number of blocks read.
-    units: 1
+    unit: 1
     sum:
       monotonic: true
       aggregation: cumulative
     attributes: [ database, table, source ]
   postgresql.commits:
     description: The number of commits.
-    units: 1
+    unit: 1
     sum:
       monotonic: true
       aggregation: cumulative
     attributes: [ database ]
   postgresql.db_size:
     description: The database disk usage.
-    units: By
+    unit: By
```

Comment:
Is the unit `By` standard? I would have expected either `B` or `bytes` for bytes.

Reply:
AFAIK we follow codes from https://ucum.org/ucum.html and By is established name there